samples/python/calibrate.py

```diff
@@ -5,11 +5,21 @@
 reads distorted images, calculates the calibration and write undistorted images
 
 usage:
-    calibrate.py [--debug <output path>] [--square_size] [<image mask>]
+    calibrate.py [--debug <output path>] [--w <width>] [--h <height>] [--t <pattern type>] [--square_size]
+    [--marker_size] [--dict_name <aruco dictionary name>] [--dict_file_name <aruco dictionary file name>] [<image mask>]
+
+usage example:
+    calibrate.py --w=6 --h=4 --t=chessboard --square_size=50 ../data/left*.jpg
 
 default values:
     --debug:    ./output/
-    --square_size: 1.0
+    --w: 4
+    --h: 6
+    --t: chessboard
```

Comment:
In case if you presume chessboard as default selection, then let's presume default width and height. They correspond to sample board image in documentation.